torch/distributed/fsdp/optim_utils.py

```diff
@@ -89,13 +88,13 @@ def _communicate_optim_state(
     otherwise (where the plus 1 comes from all-gathering the padding per rank).
 
     Args:
-        flat_param (FlatParameter): The flattened parameter.
-        flat_param_state (Dict[str, Any]): The entry in the "state" part of
-        the optimizer state dict corresponding to the flattened parameter.
+        flat_param (FlatParameter): The flattened parameter. flat_param_state
+        (Dict[str, Any]): The entry in the "state" part of the optimizer state
+        dict corresponding to the flattened parameter.
```

Comment:
Will this still display correctly after merging multiple args into the same line? I assume no?

Reply:
Good catch! I changed that argument by accident (meant to only change returns).